packages/appstore/src/modules/onboarding/__test__/empty-onboarding.spec.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { mockStore, StoreProvider } from '@deriv/stores';
+import EmptyOnboarding from '../empty-onboarding';
+
+describe('EmptyOnboarding test cases:', () => {
+    it('Should render component', () => {
```

Comment:
Can you also please add the icon test as well 🙏

Reply:
Done 😄